### PR TITLE
DLPX-63604 ssh on linux needs inactivity timeout

### PR DESCRIPTION
**Pull request by grwilson**

### CODE REVIEW

File: var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml [hunk 1]
```diff
@@ -205,6 +205,11 @@
     - { key: "AllowStreamLocalForwarding", value: "no" }
     - { key: "AllowTcpForwarding", value: "no" }
     - { key: "X11Forwarding", value: "no" }
+    #
+    # Enable an inactivity timeout so that idle connections are closed.
```
Comment (prakashsurya) [inline var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml]: IMO, a better comment would explain why we're making these settings; i.e. why do we want idle connections to be closed? Currently, this just says what we're doing, which we probably could gather from the documentation for these tunables.

Comment (grwilson) [reply var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml]: I updated the comment.

**Pull request merged**